### PR TITLE
Create pid file in initializing.

### PR DESCRIPTION
**Pull request by Krout0n**
~I make this PR `Ready for review` after I understand handling Ctrl-C with tokio because I just need to implement graceful shutdown.~ I did it. We need to implement removing pid file when failed to initialize in future.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -8,12 +9,17 @@ use std::{fs::File, io::prelude::*, path::Path};
 // const HOST_PATH: &str = "/var/lib/mackerel-agent";
 // TODO: change path as /var/lib/mackerel-agent/id
 const HOST_ID_PATH: &str = "./id";
+const PID_PATH: &str = "./pid";
```
Comment (ne-sachirou) [inline src/main.rs:12]: memo: mackerel-agent.conf.root is TODO.

File: src/main.rs [hunk 2]
```diff
@@ -8,12 +9,17 @@ use std::{fs::File, io::prelude::*, path::Path};
...
 // Register the running host or get host own id.
 async fn initialize(client: &Client, conf: &Config) -> std::io::Result<String> {
-    // if !Path::exists(Path::new(HOST_PATH)) {
-    //     std::fs::create_dir(HOST_PATH)?;
-    // }
+    if Path::new(PID_PATH).exists() {
+        panic!("Other mackerel-agent-rs process is working on!");
+    }
+
+    // todo: write the pid to pid_file.
```
Comment (ne-sachirou) [inline src/main.rs:20]: 👍

**Review by ne-sachirou — APPROVED** [review]
LGTM 🙆‍♀️

**Comment by Krout0n** [comment]
Thank you.

**Pull request merged**